### PR TITLE
Feature/#75 skip topics

### PR DESCRIPTION
**Pull request by ampasc**
It is now possible to mark Topics as skipped, like described [here](https://github.com/4minitz/4minitz/issues/75). This adaption contains

- the adaption itself ;-)
- a Migration, incl. Migration-Test
- E2E-Tests
- Extended UserDoc

### CODE REVIEW

File: server/migrations/migrate_v12.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import { MinutesCollection } from '/imports/collections/minutes_private'
+import { MeetingSeriesCollection } from '/imports/collections/meetingseries_private'
+
+export class MigrateV12 {
+
+    static _upgradeTopics(topics) {
+        topics.forEach(topic => {
+			if (topic.isSkipped === undefined) {
```
Comment (migerh) [inline server/migrations/migrate_v12.js]: Indentation

File: tests/end2end/TopicsSkip-test.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        E2EApp.gotoStartPage();
+        expect (E2EApp.isLoggedIn()).to.be.true;
+
+        aMeetingCounter++;
+        aMeetingName = aMeetingNameBase + aMeetingCounter;
+
+        E2EMeetingSeries.createMeetingSeries(aProjectName, aMeetingName);
+        E2EMinutes.addMinutesToMeetingSeries(aProjectName, aMeetingName);
+		
+		E2ETopics.addTopicToMinutes(skippedTopicName);
```
Comment (migerh) [inline tests/end2end/TopicsSkip-test.js]: Indentation

File: tests/unit/server/migrations/migrate_v12.test.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    afterEach(function () {
+        MinutesCollection.update.reset();
+        MeetingSeriesCollection.update.reset();
+    });
+
+    describe('#up', function () {
+
+        let checkTopicHasProperty = topic => {
+            expect(topic).to.have.ownProperty('isSkipped');
+			expect(topic.isSkipped).to.be.false;
```
Comment (migerh) [inline tests/unit/server/migrations/migrate_v12.test.js]: Indentation

**Comment by ampasc** [comment]
Indentation is now correctly done by spaces.

**Comment by derwok** [comment]
Hi!
I checked the Acceptance Critera - the SKIP feature works & looks really great! (Looking forward to seeing it live on our branch!).  ;-)
@migerh As this PR also touches the topicElement template, which you refactored in the meantime: could you arrange to perform the merge of this PR?

**Pull request merged**